projects/liquix/index.js

```diff
@@ -0,0 +1,63 @@
+const { getLogs } = require('../helper/cache/getLogs')
+const { covalentGetTokens } = require('../helper/token')
+const { sleep } = require("../helper/utils");
+const { getUniqueAddresses } = require('../helper/utils');
+
+const config = {
+  arbitrum: {
+    network: "arbitrum",
+    factory: "0x37215Af590CED3f50a4e3Af8BE903E4307C71C1F",
+    helper: "0x6376f97fc9bc9a0ebd03fa89eF8a653A79b41e98",
+    fromBlock: 145653947,
+  },
+};
+
+const blacklistedTokens = [
+]
+
+module.exports = {
+  doublecounted: true,
+  misrepresentedTokens: true,
+  timetravel: false,
+};
+
+async function getTokens(api, owners) {
+  let tokens = []
+  for (let i = 0; i < owners.length; i++) {
+    const owner = owners[i];
+    let new_tokens = await covalentGetTokens(owner, api, { onlyWhitelisted: true, });
```

Comment:
is there a better way to get token list? `covalentGetTokens` is expensive and we like to keep its usage to a minium

Reply:
nevermind, this method is not even used